### ISSUE TITLE
`cc-domain-management`: delete button a11y name should include the path prefix

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#1621]
## Context

In the `cc-domain-management` component, the delete buttons only contain the hostname and not the path prefix.

This means that the delete button for `example.com` and the one for `example.com/path` both have the same accessible name: `delete - example.com`.

The accessible name should be: `delete - [hostname+pathPrefix` so `delete - example.com` and `delete - example.com/path`
